pkg/pillar/cmd/zedagent/handlecontent.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2017-2018 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package zedagent
+
+// content info specific parser/utility routines
+
+import (
+	"bytes"
+	"crypto/sha256"
+
+	zconfig "github.com/lf-edge/eve/api/go/config"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"github.com/satori/go.uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+var contentInfoHash []byte
+
+// cipher context parsing routine
+func parseContentInfoConfig(ctx *getconfigContext,
+	config *zconfig.EdgeDevConfig) {
+
+	log.Infof("Started parsing content info config")
```

Comment:
A bit chatty to have this as log.Info. log.Debug is better - once there is a change log.Info is ok.

Reply:
Resolved